src/braket/registers/qubit_set.py

```diff
@@ -85,8 +82,7 @@ def map(self, mapping: dict[QubitInput, QubitInput]) -> QubitSet:
             >>> mapping = {0: 10, Qubit(1): Qubit(11)}
             >>> qubits.map(mapping)
             QubitSet([Qubit(10), Qubit(11)])
-        """
-
+        """  # noqa E501
```

Comment:
This seems to be put here to ignore the line length error for a specific line of code, but seems rather out-of-place

Reply:
multi-line docstrings have their ignores go at the end: https://docs.astral.sh/ruff/linter/#error-suppression